packages/cypress/src/schematics/cypress-project/cypress-project.ts

```diff
@@ -49,7 +49,7 @@ function updateNxJson(options: CypressProjectSchema): Rule {
   return updateJsonInTree<NxJson>('nx.json', (json) => {
     json.projects[options.projectName] = {
       tags: [],
-      implicitDependencies: [options.project],
+      implicitDependencies: options.project ? [options.project] : [],
```

Comment:
To be consistent with other projects without `implicitDependencies`, I think it would be better to not have the option at all.

Currently, you can use a `noop()` rule in the default function.

Reply:
Would you not want to add the empty tags either in this case? So just don't execute the `updateNxJson(...)` method at all if no project is specified?

Reply:
Sorry, I was mistaken... you are right we do need the `tags`. The expected result should be:

```
"my-app-e2e": {
  tags: []
},
```